server/routes/privacy-notice.js

```diff
@@ -0,0 +1,20 @@
+const description = 'The Environment Agency collects information or data about you when you use this service. This notice explains how we handle this information.'
+
+module.exports = {
+  method: 'GET',
+  path: '/privacy-notice',
+  options: {
+    description: 'Privacy notice - Flood information service - GOV.UK',
+    handler: {
+      view: {
+        template: 'privacy-notice',
+        context: {
+          pageTitle: 'Privacy notice - Flood information service - GOV.UK',
+          metaDescription: description,
+          ogDescription: description,
+          feedback: false
```

Comment:
I can't find the reason for the `feedback: false` variable in the codebase, do you know what it is there for? I think it may be unnecessary. Worth investigating

Reply:
I think, looking at LFW codebase, that the feedback var is copied across from there, as in the aftercontent partial there is an `if feedback` then show feedback block